CorsixTH/Lua/graphics.lua

```diff
@@ -199,8 +199,8 @@ function Graphics:makeGreyscaleGhost(pal)
     local g_index = 0
     local g_diff = 100000 -- greater than 3*63^2 (TH uses 6 bit colour channels)
     for j = 0, #entries do
-      local entry = entries[j]
-      local diff = (entry[1] - g)^2 + (entry[2] - g)^2  + (entry[3] - g)^2
+      local entry_j = entries[j]
```

Comment:
Almost seems like this could be `possible_nearest_entry` or just `nearest_entry`. IMO, if the `j` variable ever changes, this will lose its meaning.